keps/0004-add-testing-infrastructure.md

```diff
@@ -19,41 +19,46 @@ superseded-by:
 
 ## Table of Contents
 
-- [Table of Contents](#table-of-contents)
-- [Summary](#summary)
-- [Motivation](#motivation)
-  - [Goals](#goals)
-  - [Non-Goals](#non-goals)
-- [Proposal](#proposal)
-  - [User Stories](#user-stories)
-    - [Story 1](#story-1)
-    - [Story 2](#story-2)
-  - [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
-  - [Risks and Mitigations](#risks-and-mitigations)
-- [Graduation Criteria](#graduation-criteria)
-- [Implementation History](#implementation-history)
-- [Infrastructure Needed [optional]](#infrastructure-needed-optional)
-
-[tools for generating]: https://github.com/ekalinin/github-markdown-toc
+* [Summary](#summary)
+* [Motivation](#motivation)
+   * [Goals](#goals)
+   * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+   * [Tests](#tests)
+      * [Unit tests](#unit-tests)
+      * [Integration tests](#integration-tests)
+      * [Framework tests](#framework-tests)
+   * [Kubernetes clusters](keps/0004-add-testing-infrastructure.md#kubernetes-clusters)
+   * [CICD](#cicd)
+      * [Pull Requests](#pull-requests)
+      * [Master Branch](#master-branch)
+         * [Tests](#tests-1)
+         * [Pushes](#pushes)
+         * [Schedule](#schedule)
+         * [Base Image Change](#base-image-change)
+      * [Tags/Release](#tagsrelease)
+   * [User Stories](#user-stories)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+   * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Infrastructure Needed [optional]](#infrastructure-needed-optional)
```

Comment:
This element was removed.

Reply:
What do you mean?

Reply:
You removed that section titled `Infrastructure Needed [optional]` in the file, so the Table of Contents should also have it removed.

Reply:
It's actually still there: https://github.com/kudobuilder/kudo/blob/0a88ce75f20f498e82a234da103aaa063053bfc8/keps/0004-add-testing-infrastructure.md#infrastructure-needed-optional

I removed `### Implementation Details/Notes/Constraints [optional]`.